fluent-package/Rakefile

```diff
@@ -116,7 +116,7 @@ end
 
 def template_params(params = nil)
   config = {
-    project_name: PACKAGE_NAME,
+    project_name: PACKAGE_DIR,
```

Comment:
I think it's good timing to add a new key `package_dir`.
`package_name` vs `package_dir` are different semantic so that both of them should exist here.

Reply:
so in most cases, `project_name` should be `package_dir`.
Thanks.